package.json

```diff
@@ -54,6 +54,7 @@
   },
   "devDependencies": {
     "file-loader": "^6.2.0",
+    "prettier": "^3.2.5",
```

Comment:
Been better if we add eslint airbnb + prettier in the same time. But if this difficult  - u can do this next time.

Reply:
fix